conf/defaults.ini

```diff
@@ -1189,6 +1189,9 @@ cloudWatchDynamicLabels = true
 # New expandable navigation
 newNavigation = true
 
+# On-Behalf-Of functionality for ADX data source
+adxOnBehalfOf = true
```

Comment:
It's intended that the feature flag is enabled by default?

Reply:
It was, but we are re-thinking it now 😉 
Full discussion here: https://raintank-corp.slack.com/archives/C01LVB0CYTW/p1653360314223909

I will remove it now